tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/webview/diagram/actions/BaseChangeSplitLayoutAction.java

```diff
@@ -0,0 +1,43 @@
+package io.ballerina.plugins.idea.webview.diagram.actions;
```

Comment:
Any specific reason for not adding the copyright notices for these files?

Reply:
Updated all the files with license header :)